autopush_rs/src/server/mod.rs

```diff
@@ -297,6 +300,21 @@ impl Server {
     }
 
     fn new(opts: &Arc<ServerOptions>, tx: queue::Sender) -> Result<(Rc<Server>, Core)> {
+        // Setup Sentry logging if a SENTRY_DSN exists
+        let sentry_dsn_option = option_env!("SENTRY_DSN");
+        if let Some(sentry_dsn) = sentry_dsn_option {
+            // Spin up a new thread with a new reactor core for the sentry handler
+            thread::spawn(move || {
+                let creds = sentry_dsn
+                    .parse::<sentry::SentryCredential>()
+                    .expect("Invalid Sentry DSN specified");
+                let mut core = Core::new().expect("Unable to create core");
+                let sentry = sentry::Sentry::from_settings(core.handle(), Default::default(), creds);
+                sentry.register_panic_handler(Some(|_: &PanicInfo| -> () {}));
```

Comment:
I think that the empty callback here may mean that a panic doesn't actually get logged to stdout/stderr, but maybe it'd be good to print something briefly or run the default panic handler perhaps? (just in case)